docs/public/deploying-quarkus-app.adoc

```diff
@@ -0,0 +1,84 @@
+== Deploying a Quarkus Application on Kubernetes/Openshift cluster
+
+In this example we will be deploying a https://github.com/odo-devfiles/quarkus-ex[quarkus component] that uses GraalVM and JDK1.8+.
+
+Before you start make sure you have running openshift/kubernetes cluster and do not forget to login if it is openshift cluster.
+
+. Download the example quarkus component. 
++
+[source,sh]
+----
+ $ git clone https://github.com/odo-devfiles/quarkus-ex && cd quarkus-ex
+----
+
+. Create an quarkus odo component 
++
+[source,sh]
+----
+   $ odo create quarkus myquarkus
+   Experimental mode is enabled, use at your own risk
+
+   Validation
+    ✓  Checking devfile compatibility [195728ns]
+    ✓  Creating a devfile component from registry: DefaultDevfileRegistry [170275ns]
+    ✓  Validating devfile component [281940ns]
+
+    Please use odo push command to create the component with source deployed
+----
+
+. Create a URL in order to access the deployed component:
++
+[source,sh]
+----
+ $ odo url create --host apps-crc.testing
+  ✓  URL myquarkus-8080 created for component: myquarkus
+
+ To apply the URL configuration changes, please use odo push
+----
++
+NOTE: You must use your cluster host domain name when creating your URL.
+
+. Push the component to the cluster:
++
+[source,sh]
+----
+  $ odo push
+
+Validation
+ ✓  Validating the devfile [44008ns]
+
+Creating Kubernetes resources for component myquarkus
+ ✓  Waiting for component to start [10s]
+
+Applying URL changes
+ ✓  URLs are synced with the cluster, no changes are required.
+
+Syncing to component quarkus-july
+ ✓  Checking files for pushing [951138ns]
+ ✓  Syncing files to the component [204ms]
+
+Executing devfile commands for component myquarkus
+ ✓  Executing init-compile command "mvn compile" [3m]
+ ✓  Executing dev-run command "mvn quarkus:dev" [1s]
+
+Pushing devfile component myquarkus
+ ✓  Changes successfully pushed to component
+
+----
+
+. View your deployed application in a browser using the generated url
++
+[source,sh]
+----
+ $ odo url list
+ Found the following URLs for component myspring
+ NAME              URL                                       PORT     SECURE
+ myquarkus-8080     http://myquarkus-8080.apps-crc.testing     8080     false
+----
+
+
+Continue devloping your application and just run `odo push`, refresh your browser to view the latest changes. 
+You can also run `odo watch` to watch changes in the source code, just refreshing the browser will render the source code changes.
```

Comment:
`You can also run odo watch to watch changes in the source code. Just refreshing the browser will render the source code changes.